### PR TITLE
Update to manager

### PR DESCRIPTION
**Pull request by ljmcgann**
Fixed the manager calls to reflect changes to the object
layer code made in the last few commits.

### CODE REVIEW

**Comment by ljmcgann** [comment]
Mainn I am trying to figure out the purpose of the _retrieve_contracts thread that gets created.

File: esi_leap/manager/service.py [hunk 1]
```diff
@@ -63,28 +60,29 @@ def stop(self):
 
     def _fulfill_contracts(self):
         LOG.info("Checking for contracts to fulfill")
-        contracts = contract.Contract.get_all_by_status(
-            self._context, statuses.CREATED)
+        contracts = contract.Contract.get_all(
+            self._context, {'status': [statuses.ACTIVE, statuses.CREATED]})
```
Comment (tzumainn) [inline esi_leap/manager/service.py:64]: If a contract is active, I don't think we need to fulfill it again.

File: esi_leap/manager/service.py [hunk 2]
```diff
@@ -63,28 +60,29 @@ def stop(self):
...
+        now = timeutils.utcnow()
         for c in contracts:
-            if c.start_time and \
-               c.start_time <= timeutils.utcnow():
+            if c.start_time >= now and c.end_time <= now:
```
Comment (tzumainn) [inline esi_leap/manager/service.py:67]: Isn't the opposite of what we want? This seems to be an impossible condition:
```
now -- start time ---------- end time -- now
```
We want:
```
start time -----now------end time
```
Which means start_time <= now and now <= end_time

Comment (ljmcgann) [reply esi_leap/manager/service.py:67]: My bad. I did do a test run before but I mustve changed it before making the commit

File: esi_leap/manager/service.py (hunk 1 above)
Comment (ljmcgann) [reply esi_leap/manager/service.py]: fixed

File: esi_leap/manager/service.py (hunk 2 above)
Comment (ljmcgann) [reply esi_leap/manager/service.py]: fixed

**Comment by tzumainn** [comment]
Looks good!

**Pull request merged**